File: roms-kate_svn/MCKR_4km/InputFiles/BC_IC_2018/ini_121/makeNegLonsPos.py
import numpy as np
import netCDF4
import sys

ncfile = sys.argv[1]
nc = netCDF4.Dataset(ncfile, 'a', format='NETCDF3_CLASSIC')

lon = nc.variables['lon_rho'][:,:]
lon = (lon+360) % 360
nc.variables['lon_rho'][:,:] = lon

lon = nc.variables['lon_psi'][:,:]
lon = (lon+360) % 360
nc.variables['lon_psi'][:,:] = lon

lon = nc.variables['lon_u'][:,:]
lon = (lon+360) % 360
nc.variables['lon_u'][:,:] = lon

lon = nc.variables['lon_v'][:,:]
lon = (lon+360) % 360
nc.variables['lon_v'][:,:] = lon

nc.close()

File: roms-kate_svn/SO_4km/InputFiles/BC_IC_CMEMS_2017_2018/makeNegLonsPos.m
file = 'CMEMS_2017_2018_bdry_SO_4km.nc';

dum = nc_varget(file,'lon_rho');
dum = dum+360;
nc_varput(file,'lon_rho',dum);

dum = nc_varget(file,'lon_psi');
dum = dum+360;
nc_varput(file,'lon_psi',dum);

dum = nc_varget(file,'lon_u');
dum = dum+360;
nc_varput(file,'lon_u',dum);

dum = nc_varget(file,'lon_v');
dum = dum+360;
nc_varput(file,'lon_v',dum);



